programs/test/udp_proxy.c

```diff
@@ -73,9 +73,10 @@ int main( void )
 #endif
 #endif /* _MSC_VER */
 #else /* ( _WIN32 || _WIN32_WCE ) && !EFIX64 && !EFI32 */
-#if defined(MBEDTLS_HAVE_TIME)
+#if defined(MBEDTLS_HAVE_TIME) || (defined(MBEDTLS_TIMING_C) && !defined(MBEDTLS_TIMING_ALT))
```

Comment:
@AndrzejKurek LOL :)

(@daverodgman don't worry, just flash-backs to a previous PR!)